.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -92,8 +95,10 @@ Do the following steps to keep track of spartacussampledata releases:
     - [ ] `npm run release:smartedit:with-changelog` (needed since `3.2.0-next.0`)
     - [ ] `npm run release:qualtrics:with-changelog` (needed since `3.1.0-next.0`)
     - [ ] `npm run release:product-configurator:with-changelog` (needed since `3.1.0-next.0`)
-    - [ ] (for <3.2.0 releases set the spartacus peerDependencies manually, then)  
-      `npm run release:cdc:with-changelog` (since 3.2.0 release like any other lib with the same version as everything else. For older versions since 2.1.0-next.0 - publish under `0.<packages-version>.0` eg. `0.201.0-next.0` for first `2.1.0-next.0` release)
+    - [ ] `npm run release:cdc:with-changelog` (needed since `3.1.0-next.0`)
+      - [ ] For < 3.2.0 releases ONLY, set the spartacus peerDependencies manually, then)  
```

Comment:
There are closing `)` on this and the next line. Where is the `(`?

Reply:
Deleted.